### ISSUE TITLE
Commited autogenerated files

### ISSUE THREAD

**Issue by povilasb** [zendesk/samson#720]
`Gemfile.lock` and `vendor/` can be generated from `Gemfile`, but they were commited to git.

I consider this redundant. 

Especially, that `vendor/` size is about 27 MB.

**Comment by jonmoter** [comment]
`Gemfile.lock` can be generated, but if you delete it and regenerate it on 2 different days, you can wind up with different versions of gems.  Newer versions of gems could potentially introduce bugs.  So we run our automated tests against a checked-in Gemfile.lock, and if it passes, that means that Samson should work, as long as you use those exact gems.

It also means that if we (re)deploy a specific version of Samson, it's guaranteed to run with the exact same gem versions.

Agreed that we don't need to have the gems checked into vendor.  In the past, we had some problems grabbing gems from rubygems.org at deploy time, so we started the practice of checking them into repos.

**Comment by povilasb** [comment]
Yes, I fixed myself regarding `Gemfile.lock`.

**Comment by steved** [comment]
Is there a specific version of something you need? `Gemfile.lock`'s are generally committed for applications.

**Comment by povilasb** [comment]
No, I'm ok with the versions. The only my concern was, if committing `vendor/` wasn't redundant.

**Comment by steved** [comment]
Yeah, it's just legacy from our internal practices that we can probably remove. Git LFS might be a good match, but I haven't had enough time to play with it.
